extensions/logging-gelf/runtime/src/main/java/io/quarkus/logging/gelf/GelfConfig.java

```diff
@@ -0,0 +1,52 @@
+package io.quarkus.logging.gelf;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.RUN_TIME, name = "log.handler.gelf")
+public class GelfConfig {
+    /**
+     * Determine whether to enable the GELF logging handler
+     */
+    @ConfigItem
+    boolean enabled;
+
+    /**
+     * Hostname/IP-Address of the Logstash/Graylog Host
+     * By default it uses UDP, prepend tcp: to the hostname to switch to TCP, example: "tcp:localhost"
+     */
+    @ConfigItem(defaultValue = "localhost")
+    public String host;
+
+    /**
+     * The port
+     */
+    @ConfigItem(defaultValue = "12201")
+    public int port;
+
+    /**
+     * GELF version: 1.0 or 1.1
+     */
+    @ConfigItem(defaultValue = "1.1")
+    public String version;
+
+    /**
+     * Whether to post Stack-Trace to StackTrace field (true/false/throwable reference [0 = throwable, 1 =
+     * throwable.cause, -1 = root cause]).
+     */
+    @ConfigItem(defaultValue = "true")
+    public String extractStackTrace;
+
+    /**
+     * Whether to perform Stack-Trace filtering
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean filterStackTrace;
```

Comment:
The default is already `false` and doesn't need to be given.

Reply:
done